### DISCUSSION TITLE
Product Idea

### DISCUSSION THREAD

**Discussion by nishant400** [jhc13/mental-math-trainer#2] [Ideas]
Hi, I want to make this into a product specifically made for Indian audience who take Common aptitude test in India. i would love to collaborate on this.

**Comment by jhc13** [comment]
Hello, I currently do not have plans to turn the app into a product, and I am also busy with other matters.

However, you are free to do whatever you want with the code as long as you abide by the terms of the [license](https://github.com/jhc13/mental-math-trainer/blob/main/LICENSE).
